themes/classic/_dev/css/modules/customer.scss

```diff
@@ -1,9 +1,19 @@
-.page-authentication {
-  .no-account {
-    text-align: center;
-    font-size: 1.25rem;
-    margin: 1rem 0;
+.table-labeled {
+  th {
```

Comment:
I have not read the entire RSCSS norm, but maybe we can avoid this little duplicate by using `tr, td` here?